kitsune/search/base.py

```diff
@@ -344,6 +344,10 @@ def __getitem__(self, key):
             return self.results[0]
         return self.results
 
+    def get_filter(self):
+        """Placeholder for subclasses, should be implemented there."""
+        raise NotImplementedError
+
```

Comment:
This can be removed because all of the sub-classes of `SumoSearch` already implement `get_filter`, and in general, it's impossible to instantiate a sub-class of `SumoSearch` without implementing all of its abstract methods.